pkg/action/executor/rescheduler.go

```diff
@@ -289,3 +289,29 @@ func (r *ReScheduler) reSchedule(action *turboaction.TurboAction) (*turboaction.
 
 	return action, nil
 }
+
+// as there may be concurrent actions on the same bare pod:
+//   one action is to move Pod, and the other is to Resize Pod.container;
+// thus, concurrent control should also be applied to bare pods.
+func (r *ReScheduler) moveBarePod(pod *api.Pod, nodeName string) (*api.Pod, error) {
+	podkey := util.BuildIdentifier(pod.Namespace, pod.Name)
```

Comment:
What if user has added more than one kubernetes clusters in Turbo, and in these clusters, there's duplicate pod.namespace/pod.name?

Reply:
yes, this can be a problem in multiple clusters scenarios.

But because of similar constrains,  Kubeturbo won't work for multiple clusters during the discovery process.  So let's refactor the code to support multiple clusters in future.

Reply:
Agree. And this might be a high priority case. (A simple solution would be we add target identifier as a prefix also)